doc/src/fix_shake.rst

```diff
@@ -194,19 +207,25 @@ Restart, fix_modify, output, run start/stop, minimize info
 No information about these fixes is written to :doc:`binary restart
 files <restart>`.
 
+When used during minimization, the SHAKE or RATTLE algorithms are **not**
+applied.  Strong restraint forces are applied instead.
```

Comment:
See the "note" at the very bottom of the fix addforce doc page about fix_modify energy. If that
is the case for this new feature, the same note should appear in this section of this doc page.